modules/llm-cache/tests/kv_state_cache_benchmark_test.cc

```diff
@@ -35,8 +35,13 @@ using namespace vineyard;  //  NOLINT(build/namespaces)
 KVStateCacheManager* manager;
 
 void init() {
-  manager =
-      new KVStateCacheManager(DIMENSION, CAPACITY, LAYER, DEFAULT_BLOCK_SIZE);
+  Status status = KVStateCacheManager::MakeKVStateCacheManager(
+      manager, DIMENSION, CAPACITY, LAYER, BLOCK_SIZE, 3,
+      std::string(getenv("VINEYARD_IPC_SOCKET")));
+  if (!status.ok()) {
+    LOG(ERROR) << "Failed to create KVStateCacheManager: " << status.ToString();
+    exit(1);
```

Comment:
Use `VINEYARD_CHECK_OK(KVStateCacheManager::Make(...))` in test cases.

Reply:
OK